tests/v2/validation/rbac/globalrolesv2/globalroles_v2.go

```diff
@@ -209,3 +217,47 @@ func createDownstreamCluster(client *rancher.Client, clusterType string) (*manag
 
 	return clusterObject, steveObject, testClusterConfig, nil
 }
+
+func createGlobalRole(client *rancher.Client, inheritedClusterrole []string) (*v3.GlobalRole, error) {
+	globalRole.Name = namegen.AppendRandomString("testgr")
+	globalRole.InheritedClusterRoles = inheritedClusterrole
+	createdGlobalRole, err := rbac.CreateGlobalRole(client, &globalRole)
+	return createdGlobalRole, err
+}
+
+func createGlobalRoleAndUser(client *rancher.Client, inheritedClusterrole []string) (*management.User, error) {
+	globalRole, err := createGlobalRole(client, inheritedClusterrole)
+	if err != nil {
+		return nil, err
+	}
+
+	createdUser, err := users.CreateUserWithRole(client, users.UserConfig(), standardUser, globalRole.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	return createdUser, err
+}
+
+func crtbStatus(client *rancher.Client, crtbName string, selector labels.Selector) error {
+	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+	defer cancel()
+
+	err := wait.PollUntilContextCancel(ctx, 1*time.Second, false, func(ctx context.Context) (done bool, err error) {
```

Comment:
can you replace `1*time.Second` with `defaults.FiveSecondTimeout` OR add a new variable for 1s in the defaults package?